db/data_migration/20160211164516_republish_topical_event_about_pages_to_publishing_api.rb

```diff
@@ -0,0 +1 @@
+AboutPage.where.not(content_id: nil).each(&:publish_to_publishing_api)
```

Comment:
Two points:
* Why would this be nil? If it is, that's a problem, because they need to be published too.
* Semantically, this is a republish action, not a publish action, but this will send the requests with an `update_type` of `publish`. I think this should use this instead: https://github.com/alphagov/whitehall/blob/master/lib/data_hygiene/publishing_api_republisher.rb